examples/customUrls.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operator.openshift.io/v1
```

Comment:
I may name this file `/examples/cr-console.operator.config.yaml` or something like that. `customUrls.yaml` may not help the user understand which config we are dealing with (I've had to answer this question numerous times & agree its confusing).

Reply:
agreed